src/urllib3/contrib/pyopenssl.py

```diff
@@ -450,9 +450,12 @@ def load_verify_locations(self, cafile=None, capath=None, cadata=None):
             cafile = cafile.encode("utf-8")
         if capath is not None:
             capath = capath.encode("utf-8")
-        self._ctx.load_verify_locations(cafile, capath)
-        if cadata is not None:
-            self._ctx.load_verify_locations(BytesIO(cadata))
+        try:
+            self._ctx.load_verify_locations(cafile, capath)
+            if cadata is not None:
+                self._ctx.load_verify_locations(BytesIO(cadata))
+        except OpenSSL.SSL.Error as e:
+            raise ssl.SSLError("unable to load trusted certificates: %r" % e)
```

Comment:
Do we want a `six.raise_from(..., None)` here to remove the "error raised while handling another exception" messaging?

Reply:
I don't know:

 * it's useful to see the original exception, eg. was it an issue with cafile/capath or cadata?
 * `six.raise_from(ssl.SSLError("unable to load trusted certificates"), e)` will loose the original exception for Python 2 users
 * All other raises in this file follow this form

I'd be happy to include your change if you still think it's a good idea, I genuinely don't know!

Reply:
Oh, the point is that we don't need information about `e` twice?

Reply:
Ah it's no problem if this is the standard for this file!